src/cmakeFindModules/FindAlembic.cmake

```diff
@@ -0,0 +1,107 @@
+# variable:
+# ALEMBIC_ROOT
+# ALEMBIC_HDF5_ROOT
+# ALEMBIC_ILMBASE_ROOT
+# ALEMBIC_OPENEXR_ROOT
+#
+# HDF5 and ILMBASE should point to the root dir used to compile alembic
+#
+# This module provides variables prefixed with ABC
+# It also sets ALEMBIC_FOUND if all the libraries and include dirs were found
+#
+
+MESSAGE(STATUS "Looking for Alembic. 1.5.8")
+
+################################################################################
+# IlmBase include dir for half and ilm libraries used in alembic
+################################################################################
+
+# Alembic includes half.h for a single function "half to float", this is unfortunate
+FIND_PATH(ABC_HALF_INCLUDE_DIR half.h
+    HINTS
+    ${ALEMBIC_ILMBASE_ROOT}/include/OpenEXR
+    $ENV{ALEMBIC_ILMBASE_ROOT}/include/OpenEXR)
+
+FIND_PATH(ABC_ILMBASE_LIBS_PATH NAMES libIex.so libIex.a
+    PATHS
+        ${ALEMBIC_ILMBASE_ROOT}/lib 
+        ${ALEMBIC_ILMBASE_ROOT}/lib64
+        $ENV{ALEMBIC_ILMBASE_ROOT}/lib 
+        $ENV{ALEMBIC_ILMBASE_ROOT}/lib64
+    NO_DEFAULT_PATH)
+
+FIND_LIBRARY(ABC_ILMBASE_IEX Iex PATHS ${ABC_ILMBASE_LIBS_PATH} NO_DEFAULT_PATH)
+FIND_LIBRARY(ABC_ILMBASE_IEXMATH IexMath PATHS ${ABC_ILMBASE_LIBS_PATH} NO_DEFAULT_PATH)
+SET(ABC_ILMBASE_LIBS ${ABC_ILMBASE_IEX} ${ABC_ILMBASE_IEXMATH}) 
+
+# OpenEXR
+FIND_LIBRARY(ABC_OPENEXR_LIBS IlmImf 
+    PATHS 
+        ${ALEMBIC_OPENEXR_ROOT}/lib
+        ${ALEMBIC_OPENEXR_ROOT}/lib64
+        $ENV{ALEMBIC_OPENEXR_ROOT}/lib 
+        $ENV{ALEMBIC_OPENEXR_ROOT}/lib64
+    NO_DEFAULT_PATH)
+
+################################################################################
+# HDF5 libraries used in alembic
+################################################################################
+
+# FIXME: hdf5 should be handled by a specialized module
+FIND_PATH(ABC_HDF5_LIBS_PATH NAMES libhdf5.so libhdf5.a
+       PATHS 
+        ${ALEMBIC_HDF5_ROOT}/lib 
+        ${ALEMBIC_HDF5_ROOT}/lib64
+        $ENV{ALEMBIC_HDF5_ROOT}/lib 
+        $ENV{ALEMBIC_HDF5_ROOT}/lib64
+       NO_DEFAULT_PATH)
+FIND_LIBRARY(ABC_HDF5 hdf5 PATHS ${ABC_HDF5_LIBS_PATH})
+FIND_LIBRARY(ABC_HDF5_HL hdf5_hl PATHS ${ABC_HDF5_LIBS_PATH})
+SET(ABC_HDF5_LIBS ${ABC_HDF5} ${ABC_HDF5_HL})
+
+################################################################################
+# ALEMBIC include and library dir
+################################################################################
+
+FIND_PATH(ABC_INCLUDE_DIR Alembic/Abc/All.h
+    PATHS
+        $ENV{ALEMBIC_ROOT}/include
+        ${ALEMBIC_ROOT}/include
+    PATH_SUFFIXES
+        alembic
+    NO_DEFAULT_PATH
+)
+
+#
+# We force the use of dynamic libraries as with using the static ones we had initialization problems.
+FIND_PATH(ABC_LIBRARY_DIR libAlembicAbc.so
+    PATHS
+        ${ALEMBIC_ROOT}/lib
+        ${ALEMBIC_ROOT}/lib64
+        $ENV{ALEMBIC_ROOT}/lib
+        $ENV{ALEMBIC_ROOT}/lib64
+    NO_DEFAULT_PATH)
+FIND_LIBRARY(ABC libAlembicAbc.so PATHS ${ABC_LIBRARY_DIR})
```

Comment:
@cpichard I think the standard way to specify the lib name for FIND_LIBRARY is without the prefix `lib` and the extension `.so` (for Mac, eg would be .dylib)